backend/tests/test_cache.py

```diff
@@ -36,20 +28,3 @@ async def reset_db():
     """
     yield
     await redis.flushdb()
-
-
-async def test_insert_genres_to_cache():
-    """
-    Test that the genre cache is created and populated correctly.
-    """
-    assert not await redis.get("genres")
-    await insert_genres_to_cache(test_genre_dict)
-    assert await redis.get("genres")
-
-    genres = json.loads(await redis.get("genres"))
-
-    assert genres
-    assert len(genres) == 3
-    assert "&" in genres[2]["label"]
-    for genre in genres:
-        assert "&" not in genre["value"]
```

Comment:
We still use the `fix_genre_ampersand`, so it might be worth keeping the test, but just test directly up against that function

Something like:
```py
def test_fix_genre_ampersand():
    genres = fix_genre_ampersand(test_genre_dict)

    assert genres
    assert len(genres) == 3
    assert "&" in genres[2]["label"]
    for genre in genres:
        assert "&" not in genre["value"]
```

Reply:
Yeah, good idea! It is done